src/structs/artist.ts

```diff
@@ -0,0 +1,104 @@
+/** @module Artist Module containing structures regarding the artist. */
+
+
+import * as tag from "./tag"
+
+
+
+/** Represents an allowed song that can be used for Project Arrhythmia. */
+export class Song {
+    audioPath: string;
+    title: string;
+    romanizedTitle?: string;
+
+    constructor(audioPath: string, title: string = "Unnamed Song", romanizedTitle?: string) {
+        this.audioPath = audioPath
+        this.title = title
+        this.romanizedTitle = romanizedTitle
+    }
```

Comment:
I feel like the default values would read better if we had something along the lines of.

```ts
export class Song {
    audioPath: string;
    title: string;
    romanizedTitle?: string;

    constructor(data: Song) {
        this.audioPath = data.audioPath
        this.title = data.title ?? "Unnamed Song"
        this.romanizedTitle = data.romanizedTitle
    }
}

new Song({
    audioPath: "",
}); // { audioPath: "", title: "Unnamed Song", romanizedTitle: null }
```